### PR TITLE
Interactive Mobile menu

### PR DESCRIPTION
**Pull request by udev-21**
add interactive menu for mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,6 +245,18 @@ <h2 class="contact-title">I'm always interested in hearing<span class="newline">
...
     </section>
+    <section id="mobile-menu">
+        <div class="icon-wrapper">
+            <img src="assets/icons/close-menu-icon.svg" alt="close menu icon">
+        </div>
+        <div class="menu-list">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
```
Comment (ShoiraTa) [inline index.html]: - [ ] These should be links that take users to corresponding part of the page. Kindly add links 😉

File: index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+let menuSection = document.querySelector('#mobile-menu')
+let menu = menuSection.querySelector(".icon-wrapper");
+menu.addEventListener('click', () => {
+    menuSection.classList.add("d-none");
+});
```
Comment (ShoiraTa) [inline index.js]: - [ ] When I change the size of the screen from desktop to mobile, it automatically opens the mobile menu. However, the mobile menu should open only when the humburger button is clicked: 

https://user-images.githubusercontent.com/77038610/189679891-056f1b76-e14a-4b38-b2c0-90a4c4461519.mp4

Please make the mobile menu invisible if the humburger button was not clicked by a user. 



- [ ] Hint: To make the menu  invisible add `d-none` class to `menuSection`, then  use `toggle`  in JS to delete or add class:

```
#html
<section id="mobile-menu" class="d-none">
```

```
# JS
menu.addEventListener('click', () => {
  menuSection.classList.toggle('d-none');
});

hamburgerIcon.addEventListener('click', () => {
  menuSection.classList.toggle('d-none');
}); 
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @udev-21 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (udev-21) [reply index.html]: Got it 👌🏽

File: index.js (hunk 2 above)
Comment (udev-21) [reply index.js]: Got it thanks

File: index.html [hunk 3]
```diff
@@ -245,6 +245,18 @@ <h2 class="contact-title">I'm always interested in hearing<span class="newline">
...
+    <section id="mobile-menu" class="d-none">
+        <div class="icon-wrapper">
+            <img src="assets/icons/close-menu-icon.svg" alt="close menu icon">
+        </div>
+        <div class="menu-list">
+            <ul>
+                <li><a href="#"> Portfolio </a></li>
+                <li><a href="#"> About </a></li>
+                <li><a href="#"> Contact </a></li>
+            </ul>
```
Comment (leonardodiasb) [inline index.html]: - [ ] It is nice that you added the `<a>` tags to your menu options but it still does not redirects the user to the correct section of the page. Kindly add this functionality to your mobile menu and remember that when the user clicks on the menu options, the menu should also disappear.
 -
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️ 
 - CSS style matching the Figma template. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (udev-21) [reply index.html]: ok, got it, thanks

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**